### PR TITLE
Add first connector (api_del_doc_with_credit_block)

### PR DESCRIPTION
**Pull request by Osara-B**
## Purpose
This pull request focuses on adding the first connector (api_del_doc_with_credit_block) under the delivery and transportation module of SAP S/4 HANA.

## Checklist
- [ ] Add first connector
- [ ]  Sanitize the Open API Specification
- [ ]  Add operation ID to the Open API Specification
- [ ]  Generate client using the Open API Specification
- [ ]  Sanitize the generated client
- [ ]  Generate mock server
- [ ]  Add tests
- [ ]  Update documentation

### CODE REVIEW

File: ballerina/api_del_doc_with_credit_block/Ballerina.toml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-name = "sap.s4hana.delivery"
+name = "sap.s4hana.api_del_doc_with_credit_block"
 version = "1.0.0"
 distribution = "2201.9.0"
 authors = ["Ballerina"]
 keywords = ["Business Management/ERP", "Cost/Paid", "Vendor/SAP", "Sales and Distribution", "SD"]
-export = ["sap.s4hana.delivery"]
-repository = "https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery"
+export = ["sap.s4hana.api_del_doc_with_credit_block"]
+repository = "https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.api_del_doc_with_credit_block"
```
Comment (niveathika) [inline ballerina/api_del_doc_with_credit_block/Ballerina.toml]: Why is the repository changed?

File: ballerina/api_del_doc_with_credit_block/Module.md [hunk 2]
```diff
@@ -35,14 +35,14 @@ designed for large-scale enterprises by SAP SE.
...
+To use the `sap.s4hana.api_del_doc_with_credit_block` connector in your Ballerina application, modify the `.bal` file as follows:
 
 ### Step 1: Import the module
 
-Import the `sap.s4hana.delivery` module.
+Import the `sap.s4hana.api_del_doc_with_credit_block` module.
 
 ```ballerina
-
+import ballerinax/sap.s4hana.api_del_doc_with_credit_block as deldcwcb;
```
Comment (niveathika) [inline ballerina/api_del_doc_with_credit_block/Module.md]:
```suggestion
import ballerinax/sap.s4hana.api_del_doc_with_credit_block as delDoc;
```

File: ballerina/build.gradle [hunk 3]
```diff
@@ -24,6 +24,7 @@ description = 'Ballerina - SAP S/4HANA Delivery and Transporation APIs'
 subprojects {
     ext {
         packageOrg = "ballerinax"
+        ballerinaTomlFilePlaceHolder = new File("${project.rootDir}/build-config/resources/Ballerina.toml")
```
Comment (niveathika) [inline ballerina/build.gradle]: Remove this

File: ballerina/api_del_doc_with_credit_block/Ballerina.toml (hunk 1 above)
Comment (Osara-B) [reply ballerina/api_del_doc_with_credit_block/Ballerina.toml]: Change added in [Add PR review changes to gradle build](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/7/commits/031c361cb9b165454c9d51d179f5c52002682d85)

File: ballerina/api_del_doc_with_credit_block/Module.md (hunk 2 above)
Comment (Osara-B) [reply ballerina/api_del_doc_with_credit_block/Module.md]: [Add PR review changes to gradle build](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/7/commits/031c361cb9b165454c9d51d179f5c52002682d85)

File: ballerina/build.gradle (hunk 3 above)
Comment (Osara-B) [reply ballerina/build.gradle]: [Add PR review changes to gradle build](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/7/commits/031c361cb9b165454c9d51d179f5c52002682d85)

**Review by niveathika — DISMISSED** [review]

**Review by niveathika — DISMISSED** [review]

**Review by niveathika — DISMISSED** [review]

**Review by niveathika — APPROVED** [review]

**Pull request merged**